src/core/src/lib/services/formly.config.ts

```diff
@@ -35,6 +35,9 @@ export class FormlyConfig {
         (field.formControl?.touched || field.options.parentForm?.submitted || !!field.field.validation?.show)
       );
     },
+    showValid(field: FieldType) {
+      return field.formControl?.touched || field.options.parentForm?.submitted || !!field.field.validation?.show;
```

Comment:
@MaxKless 
Maybe needs to check `field.formControl.valid`.

Reply:
Yeah good point, fixed.